TESTING.md

```diff
@@ -1,22 +1,18 @@
 # Testing 
-End-to-end tests for CHT-Sync can be found under `tests/`. In order to run these end-to-end tests locally, follow the steps below:
+End-to-end tests for CHT Sync can be found under `tests/`. 
 
-1. Create an `.env` file with the placeholder values from `env.template`.
+Pre-requisites:
+* `Node.js` version > 20
 
-2. Install the dependencies via:
+In order to run these end-to-end tests locally, follow the steps below:
 
-```sh
-npm install
-```
+1. Install the dependencies via:
 
-3. Run the Docker containers locally:
 ```sh
-# starts: logstash, postgres, postgrest, data-generator, couchdb and dbt
-docker-compose -f docker-compose.couchdb.yml -f docker-compose.postgres.yml -f docker-compose.yml up -d
+npm ci
 ```
 
-3. Wait for every container to be up and running.
-4. Run the end-to-end tests:
+2. Run the end-to-end tests:
 
 ```sh
 # runs tests/e2e-test.spec.ts
```

Comment:
I _think_ the command to run e2e tests is `npm run test:e2e`. `npm test` right now only runs unit tests for couch2pg (and in the future, should run any unit tests that exist in the monorepo).

Reply:
You are perfectly right; thanks for pointing it out. I separated the instructions into running unit tests & running e2e tests.